web/js/components/timeline/timeline-data/info-modal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const DataPanelInfoModal = () => (
+  <div className="data-panel-info">
+    <header>
+      <p>
+        Temporal coverage of activated time-varying layers are shown in the layer coverage panel as striped blue lines in relation to the current timeline zoom level. The total temporal coverage dates are shown on the right.
+      </p>
+      <p>
+        Time-varying layers that are temporally coincident/have matching temporal coverage are shown in the timeline as a solid blue line, making it easier to find out when multiple layers have concurrent imagery available.
```

Comment:
Maybe more clear:

A solid blue line in the timeline indicates a date range where all active temporal layers have overlapping coverage, making it easier to find where imagery is concurrent between layers.